src/kdl_parser.cpp

```diff
@@ -171,5 +170,247 @@ bool treeFromUrdfModel(const urdf::ModelInterface& robot_model, Tree& tree)
     return true;
 }
 
+/*
+ * SDF to KDL
+ * bellow there are the source codes to convert SDFs information to KDL
+ */
+
+typedef std::map<std::string, sdf::ElementPtr> JointsMap;
+typedef std::map<std::string, sdf::ElementPtr> LinksMap;
+typedef std::map<std::string, std::vector<std::string> > LinkNamesMap;
+
+/**
+ * convert <axis> element to KDL::Vector
+ */
+KDL::Vector sdf_axis_to_kdl(sdf::ElementPtr sdf)
+{
+    double x, y, z;
+    std::string axis_value = sdf->GetElement("xyz")->Get<std::string>();
```

Comment:
There is a Vector type in SDF. Would not be surprised that you can do Get<SDF::Vector>